tests/test_nbr_health.py

```diff
@@ -49,26 +52,27 @@ def check_bgp_facts(hostname, host):
     if not res.has_key('stdout_lines') or u'BGP summary' not in res['stdout_lines'][0][0]:
         return "neighbor {} bgp not configured correctly".format(hostname)
 
-def test_neighbors_health(duthost, localhost, nbrhosts, eos):
+def test_neighbors_health(duthosts, localhost, nbrhosts, eos):
```

Comment:
Can we parametrize by the dut hostname similar to the one below?
https://github.com/Azure/sonic-mgmt/pull/2453

Reply:
Great suggestion! Done.